staging/src/k8s.io/client-go/util/flowcontrol/backoff_test.go

```diff
@@ -250,3 +250,68 @@ func TestBackoffWithJitter(t *testing.T) {
 
 	t.Logf("exponentially backed off jittered delays: %v", delays)
 }
+
+func TestAlternateHasExpiredFunc(t *testing.T) {
+	clock := testingclock.NewFakeClock(time.Now())
```

Comment:
any reason this is global across the test cases?

Reply:
No -- moved in 6d33dda63b656e63e9eef41194b9bbd9f65a2b13